src/utils/constans.ts

```diff
@@ -0,0 +1,10 @@
+enum SearchCityTypes {
```

Comment:
this enum have to contain only types of what you are searching for, according to the documentation. Others should be moved to separate constants